Modules/_multiprocessing/semaphore.c

```diff
@@ -740,13 +740,13 @@ static PyMethodDef semlock_methods[] = {
 
 static PyMemberDef semlock_members[] = {
     {"handle", T_SEM_HANDLE, offsetof(SemLockObject, handle), Py_READONLY,
-     ""},
+     PyDoc_STR("")},
```

Comment:
This can be just set to `NULL` I guess. It should still give an empty string in `__doc__` (I might be wrong, this needs to be validated).

Reply:
It gives None, but yeah this seems like a fair simplification